src/app/static/src/scss/style.scss

```diff
@@ -61,8 +61,8 @@ a {
   margin-left: 5px;
   margin-bottom: 5px;
   margin-top: -5px;
-  height: 20px;
-  width: 20px;
+  height: 21px;
+  width: 21px;
```

Comment:
Made some small tweaks here so that the class could be used for cross as well as the tick. Doesn't seem to have any deleterious effects on the tick.